### ISSUE TITLE
Can I encode a javascript object like this?

### ISSUE THREAD

**Issue by coodoo** [dcodeIO/ProtoBuf.js#332]
Could I do something like this?

```javascript
var obj = {foo: 123, bar: [456, 789]};
var result = capnp.encode( obj );
```

Thanks.

**Comment by dcodeIO** [comment]
Of course

**Comment by coodoo** [comment]
Mind pointing me at the right direction on how to do that?

I'm looking at [this example](https://github.com/dcodeIO/ProtoBuf.js/blob/master/examples/protoify/index.js), wondering is this the suggested way to do that? Thanks.

**Comment by dcodeIO** [comment]
In ProtoBuf.js, ever message you define has it's own runtime class. Given that, you can just create instances of these classes, providing the data the message holds as an object. Additionally, every message class has a static method `.encode(...)` that takes an object representing a message's contents and encodes it.

**Comment by coodoo** [comment]
Thanks, I had successfully run the [example](https://github.com/dcodeIO/ProtoBuf.js/blob/master/examples/protoify/index.js) and now understand how it works.

One thing to note though I did a quick test and save both the encoded `buf` and `json` into files to compare their sizes, the results were quite interesting, see the [zip file](http://cl.ly/3Y2P063P310m).

Here's how I persisted them
```
fs.writeFileSync( idx+'.buf', buf);
fs.writeFileSync( idx+'.json', JSON.stringify(sample));
```

The results showed most of the time `protobuf` is larger than their `json` counterpart, is this normal? If so, what's the *gain of using protobuf as an encoding format over the wire*? Thanks.
